cmake/luajit.cmake

```diff
@@ -75,6 +75,12 @@ if(ENABLE_ASAN)
     add_definitions(-DLUAJIT_USE_ASAN=1)
 endif()
 
+if(LUAJIT_JIT_STATUS)
```

Comment:
I'm strongly disagree that this is the right place to put this option. <cmake/luajit.cmake> file is about __LuaJIT configuration__, not Tarantool's behaviour. I suggest to move this define into <src/trivia/config.h.cmake> like it is done for `ENABLE_BACKTRACE` and other flags.